src/entities/track/common/ui/TrackCalendarHeader/UserSelect.tsx

```diff
@@ -20,6 +20,9 @@ export const UserSelect = memo(({ allowClear, userOptions, isLoading, setShouldL
     updateUserId(nextUserId);
   };
 
+  const handleFilterOption = (input: string, option?: DefaultOptionType) =>
+    (option?.label?.toString() ?? '').toLowerCase().includes(input.toLowerCase());
```

Comment:
`.toString()` не выдаст `null` или `undefined`, только которые реагируют в нулевом слиянии. Ну и при отсутствии лэйбла можно не гнать весь чейн.
```suggestion
    !option?.label || option.label.toString().toLowerCase().includes(input.toLowerCase());
```